sdk/keyvault/keyvault-certificates/review/keyvault-certificates.api.md

```diff
@@ -69,11 +71,19 @@ export interface CertificateProperties extends ParsedKeyVaultEntityIdentifier {
     readonly x509Thumbprint?: Uint8Array;
 }
 
+// Warning: (ae-forgotten-export) The symbol "CertificatesClientInterface" needs to be exported by the entry point index.d.ts
+// 
 // @public
-export class CertificatesClient {
+export class CertificatesClient implements CertificatesClientInterface {
     constructor(url: string, credential: TokenCredential, pipelineOrOptions?: ServiceClientOptions | NewPipelineOptions);
     // Warning: (ae-forgotten-export) The symbol "BackupCertificateResult" needs to be exported by the entry point index.d.ts
     backupCertificate(name: string, options?: RequestOptionsBase): Promise<BackupCertificateResult>;
+    beginCertificateCreate(name: string, certificatePolicy: CertificatePolicy, createCertificateOptions?: CreateCertificateOptions, pollerOptions?: {
+        manual?: boolean;
```

Comment:
missing option `resumeFrom` to take a previous poller/poller state

Reply:
It does allow to receive pollerOptions, which has an optional "operation" property that can be used for this purpose.

Reply:
ah, I think operation is too heavy for consumers to think much about, I'd kinda prefer to hide it from them and use `resumeFrom` to take a `CreateCertificatePoller` or a `string` directly

Reply:
oh, I understand now